<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>QUẢN LÝ BÃI XE THÔNG MINH</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>60</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Biển số xe:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputPlate">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>60</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>100</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Loại xe:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboType">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>100</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Xe Máy</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ô Tô</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnIn">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Cho xe vào</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOut">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cho xe ra &amp; Tính tiền</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>210</y>
      <width>301</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>STT</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Biển số</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Loại xe</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>961</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
